emily_landi/lib/greet.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+//ORIGINAL
+// module.exports = function(name){
```

Comment:
Keeping commented out code is understandable for what your working on (in assignments or branches) but when submitting for pull requests (or committing to master), it should be left out.